arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -11,209 +11,226 @@ import PublicPreviewBannerPartial from '../partials/_public-preview-banner-parti
 
 <PublicPreviewBannerPartial />
 
-## Prerequisites
+This guide will get you started with <a data-quicklook-from="stylus">Stylus</a>' basics. We'll cover the following steps:
+
+1. [Setting up your development environment](./stylus-quickstart#setting-up-your-development-environment)
+2. [Creating a new Stylus contract](./stylus-quickstart#creating-a-stylus-project-with-cargo stylus)
```

Comment:
This link seems to be broken now (possibly due to changing the title)

Reply:
Fixed, thanks for that!